Tasks/FtpUpload/ftputils.ts

```diff
@@ -99,6 +115,31 @@ export class FtpHelper {
         }
     }
 
+    async cleanRemoteContents(remotePath: string) {
+        tl.debug('cleaning remote directory contents: ' + remotePath);
+       
+        var that: any = this;
+        var defer: Q.Deferred<void> = Q.defer<void>();
+        this.ftpClient.list(path.normalize(remotePath), async function (err, list) {
+            try {
+                if (!err) {
+                    for (var remote of list) {
```

Comment:
add a type for "remote"?

Reply:
I haven't found any typings for the return results.  Use 'any'?